### ISSUE TITLE
[Feature] Can sglang return the complete logits？

### ISSUE THREAD

**Issue by tryedk** [sgl-project/sglang#7395]
### Checklist

- [x] 1. If the issue you raised is not a feature but a question, please raise a discussion at https://github.com/sgl-project/sglang/discussions/new/choose Otherwise, it will be closed.
- [x] 2. Please use English, otherwise it will be closed.

### Motivation

Can sglang return the complete logits with the following shape: the probability distribution of [batch, seq_len, vocab]
Something like the following:
	tensor([[[ 4.2188,  3.4219,  2.9062,  ...,  0.5625,  0.5625,  0.5625],
	         [10.4375, 13.2500, 11.1875,  ...,  3.4062,  3.4062,  3.4062],
	         [ 1.8828, 10.6250, 11.3750,  ...,  2.7188,  2.7188,  2.7188],
	         ...,
	         [12.1875, 10.6250, 10.8750,  ...,  3.9375,  3.9375,  3.9375],
	         [ 5.7188,  2.8750,  5.9375,  ...,  4.3125,  4.3125,  4.3125],
	         [ 9.6250,  6.0000,  5.0938,  ...,  5.7812,  5.7812,  5.7812]]],
	       device='cuda:0', dtype=torch.bfloat16)
The specific values above were obtained by me from the huggingface framework————
outputs = self.teacher_model(**inputs).logits

**Comment by tommyip** [comment]
Here is a patch on top of v0.5.5 to return raw logits in the `engine.generate` API:

https://github.com/beginning-ai/sglang/commit/d05d1a93824ca362c939370d45880f51eafdb609

Usage:
```py
llm = sgl.Engine(model_path="Qwen/Qwen3-0.6B")
output = llm.generate(
    ["The capital of France is", "SGLang doesn't support returning raw logits"],
    sampling_params={
        "max_new_tokens": 5,
        "top_k": 1,
    },
)
for out in output:
    logits = out["meta_info"]["output_token_logits"]
    print(f"Expected tokens: {torch.argmax(logits, axis=1)}")
    print(f"Actual tokens: {out['output_ids']}")
```
